### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by marius-schaefer**
Updated register link, dates, description, and prize tracks

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -39,15 +39,15 @@ export default function Home() {
...
+              August 6-7th, 2022 &middot; Hybrid
             </h3>
 
             <motion.button
               whileHover={{ scale: 1.1 }}
               whileTap={{ scale: 0.9 }}
               className="p-2 px-6 text-base font-medium bg-white rounded-full font-inter md:text-lg w-min"
             >
-              <Link href="/register">Register</Link>
+              <Link href="https://airtable.com/shrshwlldsgdG9GVs">Register</Link>
```
Comment (exu3) [inline pages/index.js]:
```suggestion
              <Link href="/register">Register</Link>
```

See next.config.js; it is a redirect to the Airtable link.

File: pages/index.js [hunk 2]
```diff
@@ -77,7 +74,7 @@ export default function Home() {
             <Question title="Where is this held?">
               NeoHacks is a hybrid hackathon. The in-person event will take
               place in Seattle, Washignton. Location TBD. Participants may also
-              join us virtually on Discord.
+              join us virtually on Discord from anywhere on the globe.
```
Comment (exu3) [inline pages/index.js]:
```suggestion
              join us virtually on Discord from anywhere in the world.
```

File: pages/index.js [hunk 3]
```diff
@@ -107,13 +104,14 @@ export default function Home() {
               Every project submitted will be entered in this category.
             </Card>
             <Card type={2} icon="pencil" color="red" heading="Most Creative">
-              Make your wildest ideas a reality and earn the best creative prize
-              to win a gift card for your team!
+              Make your wildest ideas a reality and earn the best creative prize!
```
Comment (exu3) [inline pages/index.js]:
```suggestion
              Make your wildest ideas a reality and earn the Most Creative prize!
```

File: pages/index.js [hunk 4]
```diff
@@ -107,13 +104,14 @@ export default function Home() {
...
             </Card>
             <Card type={2} icon="globe" color="blue" heading="Best Web App">
-              Create a web application using any technologies of choice for a
-              chance to win a gift card!
+              Create a web application using any technologies of choice. The best web app will win the Best Web App prize!
             </Card>
+            <p className="text-lg ">
```
Comment (exu3) [inline pages/index.js]:
```suggestion
            <p className="text-lg">
```

File: pages/index.js (hunk 1 above)
Comment (marius-schaefer) [reply pages/index.js]: will do

**Pull request merged**